docusaurus.config.js

```diff
@@ -7,6 +7,35 @@ if (!process.env.CI) {
     : 'fintan-contents/mobile-app-crib-notes';
 }
 
+const injector = (options) => {
+  const keys = Object.keys(options);
+  const placeHolders = keys.map((key) => new RegExp('{@inject: ?' + key + '}'));
+  return inject;
+  function inject(tree) {
+    if (tree.type === 'root' || tree.type === 'element') {
+      tree.children = tree.children.map((c) => inject(c));
+      if (tree.tagName === 'a' && tree.properties?.href) {
+        const href = decodeURI(tree.properties?.href);
```

Comment:
ここって`?`つけてあげないとエラーになるかな？`if`でチェックしてるから`undefined`になることはないと思うんだけど。

```suggestion
        const href = decodeURI(tree.properties.href);
```

Reply:
encode漏れてたから一緒に直した！

Reply:
b78a32a40454af87b9b4197202f04e263eac6d8e